docs/pages/connect-your-client/putty-winscp.mdx

```diff
@@ -27,6 +27,11 @@ You will learn how to:
 
   You should then unzip the archive and move `tsh.exe` to your `%PATH%`.
 
+  <Admonition type="note">
+    Do not place `tsh.exe` in the `System32` directory, as this can cause issues when using WinSCP.
+    You should use `%WINDIR%` (e.g. `C:\Windows`) or `%USERPROFILE%` (e.g. `C:\Users\<username>`) instead.
```

Comment:
I don't think `%USERPROFILE%` is in `Path` by default on Windows. At least not on my installation of Windows 11. When I googled "windows how to add exe to path", I didn't see people recommend this location either.

Should we recommend creating a folder in `%USERPROFILE%`, adding it to `Path` and then moving `tsh.exe` there instead? This complicates the instructions, but I don't see a better way.

FWIW, [Connect adds the folder with the bundled `tsh` to `Path`](https://github.com/gravitational/teleport/blob/master/rfd/0099-link-bundled-tsh.md#windows-1) during installation.

---

`%WINDIR%` is in system's `Path` for sure, though it's called `%SystemRoot%`, which [generally seems to be preferred](https://superuser.com/a/638324/395400).

Reply:
You're correct that `%USERPROFILE%` isn't on the path by default. With this in mind, I'm going to change the guidelines to recommend installing `tsh.exe` to `%SystemRoot%` (i.e. `C:\Windows`) by default and only using `%USERPROFILE%` if they don't have admin rights.

The other alternative is that we walk people through creating `C:\Program Files\Teleport` or similar, unzipping/moving `tsh.exe` there and then permanently adding it to the path with `setx /M PATH "%PATH%;C:\Program Files\Teleport" - but this has the risk of messing up their PATH variable globally and also requires admin rights to do, when we could just use something that's already on the PATH.

Reply:
That sounds good. It's a shame that there's no better alternative on Windows. Placing stuff in `%SystemRoot%` _feels_ wrong, but I guess it's the closest thing to something like `/usr/bin`.

Reply:
I agree, it's a mess. Also the fact that you can't natively unzip from the command line - you have to invoke PowerShell 😵‍💫